mara_dbt/commands.py

```diff
@@ -210,3 +210,42 @@ def html_doc_items(self) -> [(str, str)]:
             ('selector', _.tt[self.selector] if self.selector else None),
             ('variables', _.tt[json.dump(self.variables)] if self.variables else None)
         ]
+
+
+
+class _DbtCloudCommand(Command):
+    def __init__(self):
+        """
+        Executes a dbt command against the cloud
+        """
+        super().__init__()
+
+    def shell_command(self):
+        return (
+                (f'DBT_CLOUD_HOST={config.dbt_cloud_host()} ' if config.dbt_cloud_host() else '') +
+                (f'DBT_CLOUD_API_TOKEN={config.dbt_cloud_api_token()} ' if config.dbt_cloud_api_token() else '') +
+                (f'DBT_CLOUD_ACCOUNT_ID={config.dbt_cloud_account_id()} ' if config.dbt_cloud_account_id() else '') +
+                'dbt-cloud')
+
+
+class RunDbtJob(_DbtCloudCommand):
```

Comment:
Why not `RunDbtCloudJob`?

Reply:
That was indeed an option I was thinking about but I didn’t want to use the word “cloud” in the command name because it enforces that you run something with the SAAS solution of dbt. In the future there might be an option to run jobs through the cli command. I just wanted to make the command independent of its environmental character. When dbt should decide to implement a job on cli level, I would make the environmental configuration within mara_dbt.config.

But your right RunDbtCloudJob would be possible as well …